src/components/sound.js

```diff
@@ -16,17 +16,20 @@ module.exports.Component = registerComponent('sound', {
     volume: { default: 1 }
   },
 
+  multiple: true,
+
   init: function () {
+    var id = this.id;
     this.listener = null;
     this.sound = null;
+    this.object3DName = id ? 'sound__' + id : 'sound';
```

Comment:
We're just getting the attribute name. Is there an easier way? Like `this.name` or `this.fullName`.